packages/td-bank-integration/src/td-online-mart.ts

```diff
@@ -32,3 +34,16 @@ interface CreateTokenError {
     type: string;
     message: string;
 }
+
+/* eslint-disable @typescript-eslint/naming-convention */
+export interface TdOnlineMartThreeDSErrorBody {
+    errors?: Array<{ code: string }>;
+    three_ds_result?: {
+        formUrl: string;
+        threeDSSessionData: string;
+        creq: string;
+    };
+}
+/* eslint-enable @typescript-eslint/naming-convention */
```

Comment:
[optional] It might be better to create another interface for error object

```suggestion
interface TdOnlineMartThreeDSError {
    code: string;
}

/* eslint-disable @typescript-eslint/naming-convention */
export interface TdOnlineMartThreeDSErrorBody {
    errors?: TdOnlineMartThreeDSError[];
    three_ds_result?: {
        formUrl: string;
        threeDSSessionData: string;
        creq: string;
    };
}
/* eslint-enable @typescript-eslint/naming-convention */
```

Reply:
I don't think that we need to do this. Because this interface won't be used in other places and hasn't difficult structure to split it to another interface.